tests/test_config_section_virt.py

```diff
@@ -112,14 +112,24 @@ def test_validate_unencrypted_password(self):
         result = self.virt_config._validate_unencrypted_password('password')
         self.assertIsNone(result)
 
-    def test_validate_unicode_unencrypted_password(self):
+    @patch('virtwho.config.requests')
+    def test_validate_unicode_unencrypted_password(self, _mock_requests):
         """
         Test of validation of password that is not encrypted and it contains some
         UTF-8 string.
+        Passes or fails based on python-requests version
         """
         self.init_virt_config_section()
         self.virt_config['password'] = 'Příšerně žluťoučký kůň pěl úděsné ódy.'
+        # Version that can't handle unicode
+        _mock_requests.__version__ = '2.19.0'
         result = self.virt_config._validate_unencrypted_password('password')
+        self.assertEqual(result,
+            ('error', "Value: Příšerně žluťoučký kůň pěl úděsné ódy. of option 'password': is not in latin1 encoding."
+                  " That is not allowed on this system."))
```

Comment:
@wottop BTW: "Příšerně žluťoučký kůň pěl úděsné ódy" means "The horribly yellowish horse sang terrible odes" in Czech :smiley: and it is widely used for testing encoding in Czech Republic.

Reply:
Thanks for letting me know. I copied it from another test. ;)